sessionize/lib/src/commonMain/kotlin/co/touchlab/sessionize/api/NotificationsApiImpl.kt

```diff
@@ -0,0 +1,26 @@
+package co.touchlab.sessionize.api
+
+import co.touchlab.sessionize.platform.createLocalNotificationOnPlatform
+import co.touchlab.sessionize.platform.cancelLocalNotificationOnPlatform
+import co.touchlab.sessionize.platform.initializeNotificationsOnPlatform
+import co.touchlab.sessionize.platform.deinitializeNotificationsOnPlatform
+
+object NotificationsApiImpl : NotificationsApi {
+
+    override fun createLocalNotification(title:String, message:String, timeInMS:Long, notificationId: Int) {
```

Comment:
I don't think this pattern is quite what we want, with an interface that is just implemented by an expect/actual wrapper. Once we've created the interface in common, android and iOS can each implement it (as an interface for android, a protocol for iOS) and pass it into serviceRegistry without the expect/actual middleman.

Reply:
I'm not sure I understand. Maybe we can discuss this tomorrow.